### ISSUE TITLE
Resources referenced inside ocw-www are deleted upon gdrive sync

### ISSUE THREAD

**Issue by HussainTaj-arbisoft** [mitodl/ocw-studio#1814]
### Steps to Reproduce

1. Create/Select a course site.
2. Open the GDrive folder for this site, and upload a test file (say `example.txt`).
3. Navigate to the course's resources tab and hit `Sync w/ Google Drive`.
4. Once the sync is successful, publish the course to live.
5. Open ocw-www website. Create a Resource Collection, and add `example.txt` to it.
6. Delete `example.txt` from GDrive folder.
7. Navigate back to the course's resources tab and hit `Sync w/ Google Drive`.

### Expected Behavior

Drive sync for the course is unsuccessful. Deleting `example.txt` failed. The error states that `example.txt` is being used.

### Actual Behavior

The resource is deleted successfully.

### Related Issues (Additional Information)

In https://github.com/mitodl/ocw-studio/issues/1754 a fix was published. The fix works for active starters only. ocw-www starter is kept inactive in prod and rc.

**Comment by pdpinch** [comment]
I don't think we should limit this check to active starters. Can that be changed?

**Comment by HussainTaj-arbisoft** [comment]
> I don't think we should limit this check to active starters. Can that be changed?

Yes, it can.
